wp-content/plugins/tw-endpoint-helper/includes/class-peh-alias-object.php

```diff
@@ -96,6 +96,9 @@ public function get_rest_route() {
 				case 'category':
 					$prefix = 'categories';
 					break;
+				case 'person':
```

Comment:
Could we use `get_taxonomy` to get the custom taxonomy object and use its `rest_base` property value as the prefix? Could this be done for all our custom taxonomies?

Reply:
@rpeterman-gp i will take a look at this, i think that getting post type and taxonomy from the base and  using the rest base prefix will be a solution to avoid managing this as a hardcoded path.

Reply:
Ok, new code pushed. Please check it when you have time @rpeterman-gp